MapboxNavigation/TopBannerViewController.swift

```diff
@@ -0,0 +1,433 @@
+import Foundation
+import MapboxCoreNavigation
+import MapboxDirections
+
+
+@objc protocol TopBannerViewControllerDelegate: StatusViewDelegate {
+    @objc optional func topBanner(_ banner: TopBannerViewController, didSwipeInDirection direction: UISwipeGestureRecognizer.Direction)
```

Comment:
shouldn't these be public?

Reply:
Not in protocol definitions. I did change the protocol itself to be public, however.